pkg/registry/apis/dashboard/legacy/token.go

```diff
@@ -12,16 +12,16 @@ type continueToken struct {
 	orgId  int64
 	id     int64  // the internal id (sort by!)
 	folder string // from the query
-	size   int64
+	bytes  int64  // information, not a query
```

Comment:
what is this used for? I see it encoded into the token but not parsed back out

Reply:
The paging strategy depends on the byte size.  I added it to the token so we can see what it is actually doing -- but it is not used as part of the query

informational debugging :)

Reply:
I'd like to either remove it or properly support parsing it, so that the encoding is clear and symmetric